teachertool/src/transforms/initNewProjectResults.ts

```diff
@@ -0,0 +1,12 @@
+import { stateAndDispatch } from "../state";
+import { setEvalResultsPending } from "./setEvalResultsPending";
+import * as Actions from "../state/actions";
+
+
+export function initNewProjectResults() {
+    const { state: teacherTool, dispatch } = stateAndDispatch();
+    setEvalResultsPending({ clearAllEntries: true });
+    for (const result of Object.keys(teacherTool.evalResults)) {
+        delete teacherTool.evalResults[result].notes
```

Comment:
**Important**: Never modify the state object directly. Changes must be made in the reducer only. The UI will not update in response to this change. It may be ignored entirely.

Reply:
This is a ship blocker.

Reply:
Can you make an action for this instead?

Reply:
Can I iterate through the notes like in the for loop in the reducer?

Reply:
Iterating is ok, but you must not modify the current `state` object. You can build a new set of `evalResults` and assign it to the copy being generated in the reducer.